scripts/common.py

```diff
@@ -233,7 +233,7 @@ def validate_keyset(jwks_dict) -> Tuple[bool, List[Issue]]:
         keyset_issues.extend(issues)
 
     errors = [issue for issue in keyset_issues if issue.type.level == IssueLevel.ERROR]
-    keyset_is_valid = at_least_one_valid_keyset and len(errors) == 0
+    keyset_is_valid = at_least_one_valid_keyset
```

Comment:
Question -- in scenario 2), this script will still report the full set of keyset validation issues, though it will now be `valid` (assuming there is a single valid SHC-issuing key). Should we be clearing / silencing those issues, which may be unrelated (or may not)?